File: app/src/main/java/code/name/monkey/retromusic/helper/PlayPauseButtonOnClickHandler.java
package code.name.monkey.retromusic.helper;

import android.view.View;


public class PlayPauseButtonOnClickHandler implements View.OnClickListener {
    @Override
    public void onClick(View v) {
        if (MusicPlayerRemote.isPlaying()) {
            MusicPlayerRemote.pauseSong();
        } else {
            MusicPlayerRemote.resumePlaying();
        }
    }
}

File: app/src/main/java/code/name/monkey/retromusic/helper/PlayPauseButtonOnClickHandler.kt
package code.name.monkey.retromusic.helper

import android.view.View

class PlayPauseButtonOnClickHandler : View.OnClickListener {
    override fun onClick(v: View) {
        if (MusicPlayerRemote.isPlaying) {
            MusicPlayerRemote.pauseSong()
        } else {
            MusicPlayerRemote.resumePlaying()
        }
    }
}
